e2e/process-services/widgets/number_widget.e2e.ts

```diff
@@ -0,0 +1,109 @@
+import { LoginPage } from '../../pages/adf/loginPage';
+import { TasksPage } from '../../pages/adf/process_services/tasksPage';
+import { Widget } from '../../pages/adf/process_services/widgets/widget';
+
+import CONSTANTS = require('../../util/constants');
+import TestConfig = require('../../test.config');
+import resources = require('../../util/resources');
+
+describe('Number widget', () => {
+
+    let loginPage = new LoginPage();
+    let processUserModel;
+    let taskPage = new TasksPage();
+    let widget = new Widget();
+    let alfrescoJsApi;
+    let appsActions = new AppsActions();
+    let appModel;
+    let app = resources.Files.WIDGET_CHECK_APP.NUMBER;
+    let deployedApp, process;
+
+    beforeAll(async (done) => {
+        let users = new UsersActions();
+
+        alfrescoJsApi = new AlfrescoApi({
+            provider: 'BPM',
+            hostBpm: TestConfig.adf.url
+        });
+
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        processUserModel = await users.createTenantAndUser(alfrescoJsApi);
+
+        await alfrescoJsApi.login(processUserModel.email, processUserModel.password);
+        appModel = await appsActions.importPublishDeployApp(alfrescoJsApi, resources.Files.WIDGET_CHECK_APP.file_location);
+
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        deployedApp = appDefinitions.data.find((currentApp) => {
+            return currentApp.modelId === appModel.id;
+        });
+        process = await appsActions.startProcess(alfrescoJsApi, appModel, app.processName);
+        loginPage.loginToProcessServicesUsingUserModel(processUserModel);
+        done();
+    });
+
+    beforeEach(() => {
+        let urlToNavigateTo = `${TestConfig.adf.url}/activiti/apps/${deployedApp.id}/tasks/`;
+        browser.get(urlToNavigateTo);
+        taskPage.filtersPage().goToFilter(CONSTANTS.TASK_FILTERS.MY_TASKS);
+        taskPage.formFields().checkFormIsDisplayed();
+    });
+
+    afterAll(async (done) => {
+        await alfrescoJsApi.activiti.processApi.deleteProcessInstance(process.id);
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        await alfrescoJsApi.activiti.adminTenantsApi.deleteTenant(processUserModel.tenantId);
+        done();
+    });
+
+    it('[C269111] Number Widget - General Properties', () => {
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        expect(widget.numberWidget().getNumberFieldLabel(app.FIELD.number_general)).toContain('Number General');
+        expect(widget.numberWidget().getPlaceholder(app.FIELD.number_general)).toContain('Type a number');
+
+        widget.numberWidget().setFieldValue(app.FIELD.number_general, 2);
+        expect(taskPage.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+    });
+
+    it('[C274702] Number Widget - Advanced and visibility properties', () => {
```

Comment:
Could you change the name to : Should be able to set advanced and visibility properties for Number Widget ?